docs/enterprise/scale-capacity.md

```diff
@@ -36,6 +36,10 @@ To scale your capacity you need:
 
 6. Confirm your tier by viewing the overview tab. The current pricing tier is listed.
 
+## Known issues
```

Comment:
I suggest renaming this to 'Considerations and limitations'.

Reply:
making these changes